specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/jobs.json

```diff
@@ -0,0 +1,539 @@
+          "type": "string"
+        },
+        "localReference": {
+          "description": "Location of data inside the container process.",
+          "type": "string"
+        },
+        "mode": {
+          "$ref": "#/definitions/DataBindingMode"
+        }
+      }
+    },
+    "ScaleConfiguration": {
+      "type": "object",
+      "properties": {
+        "instanceCount": {
+          "format": "int32",
+          "description": "Number of instances.",
+          "type": "integer"
+        },
+        "instanceType": {
+          "type": "string"
+        },
+        "resourceRequirements": {
+          "type": "string"
+        },
+        "isReentrant": {
+          "type": "boolean"
+        }
+      }
+    },
+    "ComputeBinding": {
+      "description": "Compute binding definition.",
+      "type": "object",
+      "properties": {
+        "computeId": {
+          "description": "ID of the compute resource.",
+          "type": "string"
+        },
+        "nodeCount": {
+          "format": "int32",
+          "description": "Number of nodes.",
+          "type": "integer"
+        }
+      }
+    },
+    "ScheduleConfiguration": {
+      "type": "object",
+      "properties": {
+        "cron": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "CodeJob": {
+      "description": "Code Job definition",
+      "type": "object",
+      "properties": {
+        "status": {
+          "$ref": "#/definitions/JobStatus"
+        },
+        "codeConfiguration": {
+          "$ref": "#/definitions/CodeConfiguration"
+        },
+        "environmentConfiguration": {
+          "$ref": "#/definitions/EnvironmentConfiguration"
+        },
+        "dataBindings": {
+          "description": "Mapping of data bindings used in the job.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/DataBinding"
+          },
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "id": {
+          "description": "Name and identifier for a job.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "experimentName": {
+          "description": "The name of the experiment the job belongs to. If not set, the job is placed in the \"Default\" experiment.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "properties": {
+          "description": "The job property dictionary. Properties can be added, but not removed or altered.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "create",
+            "read",
+            "update"
+          ]
+        },
+        "tags": {
```

Comment:
>tags [](start = 9, length = 4)

Tags are part of top level ARM object. Do you need separate tagging in resource properties?
All the Governance scenario like policy uses top level tags.

Reply:
We use separate tagging in our MachineLearning objects (we have several scenarios that use them), and we can't remove them.

Reply:
Can we rename these to specific tags something like MLTags? This will cause confusion otherwise.

Reply:
This is a proxy resource - I'm not seeing tags on the top level ARM proxy resource entity. Unfortunately, we can't rename these, since this naming is already used across our product.

Reply:
Tags has a special purpose in Azure control plane services; customer understands tags have special meaning and this needs to be uphold. Please start and email thread and explain the reasoning of why the interface can not be adapted.